docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -472,6 +472,17 @@ When moving modules between collections
 
 See :ref:`Migrating content to a different collection <migrate_to_collection>` for complete details.
 
+Generally we do not object to moving content between collections, or moving content from collections included in Ansible to collections outside the Ansible package, as long as semantic versioning is not violated. More precisely, replacing content by redirects is only a minor change if the destination collection is the dependency of the source collection of the move. (See :ref:`coll_dependencies` for adding new dependencies to collections included in Ansible.)
```

Comment:
```suggestion
Generally, we do not object to moving content between collections or moving content from collections included in Ansible to collections outside the Ansible package, as long as semantic versioning is not violated. More precisely, replacing content with redirects is only a minor change if the destination collection is a dependency of the original collection and both collections are included in Ansible. (See :ref:`coll_dependencies` for more information about adding new dependencies to collections included in Ansible.)
```

Reply:
Should we also explicitly state "Content must not be removed from collections in minor or patch releases"? We recently had that problem with ansible.posix.

Reply:
Generally removing content is always a breaking change unless you add the moved to collection as a dependency. In that case, removal is fine for a minor release, but I don't think it's OK for a bugfix release.

Reply:
I'll commit the remainder of your suggestion now, for that I replicate it here:
```suggestion
Generally, we do not object to moving content between collections or moving content from collections included in Ansible to collections outside the Ansible package, as long as semantic versioning is not violated. More precisely, replacing content with redirects is only a minor change if the destination collection is the dependency of the original collection. (See :ref:`coll_dependencies` for more information about adding new dependencies to collections included in Ansible.)
```